frontend/src/layout/navigation-3000/components/ProjectTree/ProjectTree.tsx

```diff
@@ -51,6 +51,37 @@ export function ProjectTree({ contentRef }: { contentRef: React.RefObject<HTMLEl
     const notDraggableIds: string[] = ['project', 'project/Explore', 'project/Create new', 'project/Unfiled']
     const notDroppableIds: string[] = ['project', 'project/Explore', 'project/Create new']
 
+    const handleCreateFolder = (parentPath?: string): void => {
+        const promptMessage = parentPath ? `Create a folder under "${parentPath}":` : 'Create a new folder:'
+        const folder = prompt(promptMessage, '')
+        if (folder) {
+            const escapedFolderName = escapePath(folder)
+            const parentSplits = parentPath ? splitPath(parentPath) : []
+            const newPath = joinPath([...parentSplits, escapedFolderName])
```

Comment:
... but `joinPath` already calls `escapePath` on everything passed through it. Why are we calling it twice on the folder?